Cleaned up parameter group definitions for displayPort.

mikeller opened the pull request.

etracer65 commented:
I'm not sure I understand why the module in the `pg` directory is named `displayport_msp`. It's used in the max7456 driver as well so it seems a bit confusing. Maybe it should just be `pg/display port.c` or maybe `pg/displayportprofile.c`.

mikeller commented:
@etracer65: That's because I first wasn't aware that the struct is used in two parameter groups, and added `displayPortProfileMax7456` only later on, without thinking to change the name. Fixed.
